tests/control/DirectorTest.php

```diff
@@ -402,6 +402,9 @@ public function testUnmatchedRequestReturns404() {
 	}
 
 	public function testIsHttps() {
+		if(!TRUSTED_PROXY) {
```

Comment:
Seems like the TestSkipped string is backwards? !TRUSTED_PROXY means untrusted proxy, but test says test is skipped if !untrusted proxy.